CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
>Wouldn't a PR changing -J to default (with -j1 as the opt-out) make sense?

Maaaybe, I think I'd be +1 on that, but I suspect it might be contentious...